bin/gem2rpm

```diff
@@ -45,6 +60,11 @@ end
 srpmdir = nil
 specfile = nil
 if options[:srpm]
+  unless File.exist?(gemfile)
```

Comment:
Hi, thanks for the reviewing.
Above condition is such as below case. The case that user specified not gem file but gem name. 1st argument is gem name with --fetch option, but otherwise gem file path. I think that this specification causes user to mistake.

$ gem2rpm --srpm gem2rpm
Invalid GEMFILE

{:print_template_file=>nil, :template_file=>nil, :output_file=>nil, :local=>false, :srpm=>true, :deps=>false, :nongem=>false, :doc_subpackage=>true, :fetch=>false, :directory=>nil, :args=>["gem2rpm"]}